pkg/controller/job/job_controller_test.go

```diff
@@ -917,15 +919,61 @@ func TestAddPodOrphan(t *testing.T) {
 	}
 }
 
+func TestSuspendJob(t *testing.T) {
```

Comment:
Can we also test a Job that is created in suspended status?

Reply:
Done.